src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/GatewayOverviewOrchestratorTests.cs

```diff
@@ -173,7 +173,7 @@ public void ProcessViewModelOnError_process_view_model_correctly_GatewayReviewSt
             viewModelOnError.ErrorTextGatewayReviewStatus.Should().Be(errorMessage);
         }
 
-        [TestCase(GatewayReviewStatus.Clarification, "OptionAskClarificationText", "Error - Clarification")]
+        [TestCase(GatewayReviewStatus.AskForClarification, "OptionAskClarificationText", "Error - Ask forClarification")]
```

Comment:
Missing a space in the error message?

Reply:
I have placed a space in the error message